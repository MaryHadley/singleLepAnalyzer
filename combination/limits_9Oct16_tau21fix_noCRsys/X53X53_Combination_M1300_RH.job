universe = vanilla
Executable = X53X53_Combination_M1300_RH.sh
Should_Transfer_Files = YES
WhenToTransferOutput = ON_EXIT
Notification = Error
request_memory = 3072
Output = X53X53_Combination_M1300_RH.out
Error = X53X53_Combination_M1300_RH.err
Log = X53X53_Combination_M1300_RH.log
Queue 1