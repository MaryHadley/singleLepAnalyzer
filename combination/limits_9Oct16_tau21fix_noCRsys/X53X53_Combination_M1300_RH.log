000 (1297372.000.000) 10/10 19:27:08 Job submitted from host: <192.168.63.249:41533>
...
001 (1297372.000.000) 10/10 19:27:18 Job executing on host: <192.168.63.238:41168>
...
006 (1297372.000.000) 10/10 19:27:27 Image size of job updated: 1
	3  -  MemoryUsage of job (MB)
	2264  -  ResidentSetSize of job (KB)
...
006 (1297372.000.000) 10/10 19:32:28 Image size of job updated: 283292
	161  -  MemoryUsage of job (MB)
	164256  -  ResidentSetSize of job (KB)
...
006 (1297372.000.000) 10/10 20:12:32 Image size of job updated: 284104
	162  -  MemoryUsage of job (MB)
	165008  -  ResidentSetSize of job (KB)
...
005 (1297372.000.000) 10/10 20:15:41 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:47:58, Sys 0 00:00:03  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:47:58, Sys 0 00:00:03  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1633  -  Run Bytes Sent By Job
	323  -  Run Bytes Received By Job
	1633  -  Total Bytes Sent By Job
	323  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :     1097        1   3824816
	   Memory (MB)          :      162     3072      3072
...
