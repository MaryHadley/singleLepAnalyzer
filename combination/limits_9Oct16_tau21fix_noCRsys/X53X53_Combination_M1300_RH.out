/var/lib/condor/execute/dir_15002
Total number of likelihood evaluations: 55000101
Total number of likelihood evaluations: 5500101
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1300_RH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1300_RH/quant-toys:0-sig-ed51d2fe33.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1300_RH/quant-data-sig-613ebec633.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1300_RH
