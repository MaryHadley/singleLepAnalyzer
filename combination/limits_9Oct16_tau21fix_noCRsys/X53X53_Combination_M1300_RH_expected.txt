# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.069128   0.034968    0.14389  0.048325    0.10232
